daml-script/daml3/Daml/Script/Internal.daml

```diff
@@ -0,0 +1,110 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE AllowAmbiguousTypes #-}
+
+module Daml.Script.Internal where
```

Comment:
Perhaps explicit export list? `Daml.Script` will stop it being exported at that level, but someone could import `Daml.Script.Internal`, and use `dangerCast` here, which would be very bad.

Reply:
I'll not export it, that way the only way to get it is to generate your own LF. LFConversion checks shouldn't be needed